include/core/ib_particle.h

```diff
@@ -284,9 +281,11 @@ class IBParticle
   double rolling_friction_coefficient;
   // The particle mass.
   double mass;
+  // The particle volume.
+  double volume;
   // The surface energy of the particle
   double surface_energy;
-  // The particle rotational inertia. It is uniform in all directions.
+  // The particle moment of inertia in its reference frame
```

Comment:
So, this is the the inertia matrix of the particle in it`s frame of reference. If I understand, this a diagonal matrix right? 

```suggestion
  // The inertia matrix in the particle frame of reference
```